prboom2/src/m_menu.c

```diff
@@ -3313,7 +3313,7 @@ static const char *gen_compstrings[] =
   "9/Boom",
   "10/LxDoom",
   "11/MBF",
-  "12/PrBoom 2.03b",
+  "12/PrB 2.03b",
```

Comment:
Wait a minute. This is inconsistent. Could you abbreviate the other "prboom" strings as well?

Reply:
Sure thing. In retrospect, I could've abbreviated every instance as well as PrB+.